R/utilities-simulation-results.R

```diff
@@ -63,10 +66,7 @@ getOutputValues <- function(simulationResults, quantitiesOrPaths, individualIds
 
     for (individualIndex in seq_along(individualIds)) {
       individualId <- individualIds[individualIndex]
-      vals <- simulationResults$getValuesByPath(path = path, individualIds = individualId)
-      if (is.null(vals)) {
-        next
-      }
+      vals <- simulationResults$getValuesByPath(path = path, individualIds = individualId, stopIfNotFound)

```

Comment:
e.g. here you could do the check. AT that stage, you only have to deal with na no?